### ISSUE TITLE
TypeError when using make_classification function from sklearn.datasets

### ISSUE THREAD

**Issue by khoadng** [Almenon/AREPL-vscode#421]
**Describe the bug**
Failed to call make_classification, normal exec working fine.

**To Reproduce**

1) pip install "scikit-learn==0.24.2"
2) Run the code below

```python
from sklearn.datasets import make_classification

x, y = make_classification(n_samples=10000, weights=(0.9,0.1))

print(x.shape)
```

**Expected behavior**
No error occurred.

**Other Information (please complete the following information):**
 - OS: Windows 10 (Build 19042)
 - Python 3.8.7
 
**Additional context**

Error
```
Traceback (most recent call last):
  line 4, in <module>
  File "v:\01_Work\AI\MI\practical_deep_learning\.venv\lib\site-packages\sklearn\utils\validation.py", line 63, in inner_f
    return f(*args, **kwargs)
  File "v:\01_Work\AI\MI\practical_deep_learning\.venv\lib\site-packages\sklearn\datasets\_samples_generator.py", line 244, in make_classification
    y[flip_mask] = generator.randint(n_classes, size=flip_mask.sum())
  File "v:\01_Work\AI\MI\practical_deep_learning\.venv\lib\site-packages\numpy\core\_methods.py", line 47, in _sum
    return umr_sum(a, axis, dtype, out, keepdims, initial, where)
TypeError: int() argument must be a string, a bytes-like object or a number, not '_NoValueType'

```

**Comment by khoadng** [comment]
Might be related to #404

**Comment by Almenon** [comment]
Yeah, probably same issue :(

**Comment by khoadng** [comment]
More errors with the same type.

```python
import numpy as np

arr = np.array((1,2,3))
print(arr.min()) # error
print(arr.max()) # error
print(arr.sum()) # error
```
I'm seeing a pattern here

**Comment by Almenon** [comment]
@khoadng the code you just posted works in AREPL with python 3.9 FYI. Possibly sklearn too, I haven't tested.

**Comment by khoadng** [comment]
The issue with sklearn seems to be resolved in python 3.9 so I will close this issue.
